packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.controller.js

```diff
@@ -503,12 +525,7 @@ export default class OverTheBoxDetailsCtrl {
       })
       .$promise.then((devices) => {
         this.device = devices;
-        this.kpiInterfaces = devices.networkInterfaces
-          .filter((netInterface) => netInterface.gateway != null)
-          .map((netInterface) =>
-            netInterface.device ? netInterface.device : netInterface.name,
-          );
-
+        this.kpiInterfaces = devices.networkInterfaces;
```

Comment:
```suggestion
        this.kpiInterfaces = devices.networkInterfaces
          .filter((netInterface) => netInterface.gateway != null)
          .map((netInterface) =>
            netInterface.device ? netInterface.device : netInterface.name,
          );
```
This chart doesn't work if you apply this change. Because the this.kpiInterfaces should be an array not an array of object

for const filteredDown = inStats.filter(
          (d) => this.kpiInterfaces.indexOf(d.tags.interface) > -1,
        ); 

and same for upload

Reply:
https://github.com/ovh/manager/pull/11525